BuildForSDG/team-271-backend
# setup django project and add database config file

## Description
- Setup a new django project
- Install relevant dependencies
- Create a requirements.txt file for the dependencies 

## How Has This Been Tested?
- Clone the repo and check into branch *ch-setup-django-172636621*
- Activate the virtual environment and run *pip install -r requirements.txt*
- Create a *.env* file from the *.env.example* file and enter the database particulars of your PostgreSQL database that you've created locally
- Run *python manage.py makemigrations*
- Run *python manage.py migrate*
- Run *python manage.py runserver*


## Checklist:
